build-utils/language-c/src/main/java/org/apache/plc4x/language/c/CLanguageTemplateHelper.java

```diff
@@ -178,33 +178,44 @@ public boolean requiresPointerAccess(ComplexTypeDefinition typeDefinition, Strin
      */
     @Override
     public String getLanguageTypeNameForTypeReference(TypeReference typeReference) {
+        System.out.println("Type Reference " + typeReference.getClass());
```

Comment:
```suggestion
```
Debug statement I guess (We might add more logging to the whole language helper in the future I reckon).